run_slam_rclcpp_node:
  ros__parameters:
    vocab_file_path: /home/atas/colcon_ws/src/vox_nav/vox_nav_openvslam/config/orb_vocab.fbow
    setting_file_path: /home/atas/colcon_ws/src/vox_nav/vox_nav_openvslam/config/zed_stereo_config.yaml
    mask_img_path: ""
    map_db_path : /home/atas/large_map.msg
    debug_mode: true
    eval_log: true
    write_map_info: true
    map_info_path: "/home/atas/large_map_info.yaml"
run_localization_rclcpp_node:
  ros__parameters:
    vocab_file_path: /home/atas/colcon_ws/src/vox_nav/vox_nav_openvslam/config/orb_vocab.fbow
    setting_file_path: /home/atas/colcon_ws/src/vox_nav/vox_nav_openvslam/config/zed_stereo_config.yaml
    mask_img_path: ""
    prebuilt_map_path : /home/atas/slope_map.msg
    debug_mode: true
    eval_log: true
    enable_mapping_module: false    
    enable_pangolin_viewer: true
    map_datum:
      latitude: 49.89998651126992
      longitude: 8.90004175541922
      altitude: 0.6342219080730283
      quaternion:
        x: -0.0002674302377596963
        y: -7.804134265664371e-05
        z: 0.7068408908205831
        w: 0.7073725167509558
